externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/schema/RowSet.scala

```diff
@@ -102,11 +102,7 @@ object RowSet {
       case _ =>
```

Comment:
we can remove the `case StringType =>` branch,

and simply do

```
case _ =>
        val values = getOrSetAsNull[java.lang.String](rows, ordinal, nulls, "")
        TColumn.stringVal(new TStringColumn(values, nulls))
```

Reply:
updated